python/lsst/cp/verify/verifyPtc.py

```diff
@@ -194,6 +196,15 @@ def verify(self, calib, statisticsDict, camera=None):
             verify['GAIN'] = bool(diffGain < self.config.gainThreshold)
             verify['NOISE'] = bool(diffNoise < self.config.noiseThreshold)
 
+            # Check that the noises measured in cpPtcExtract do not evolve
+            # as a function of flux by measuring their correlation with
+            # signal level.
+            signals = calib.rawMeans[ampName]
+            noiseList = calib.noiseList[ampName]
+            params = np.polyfit(signals, noiseList, deg=1)
+            maxNoiseDiff = (signals[-1] - signals[0])*params[0]
```

Comment:
This is correct as written.  It's measuring the amount of noise change attributable to flux changes over the set of input fluxes.  This should be small relative to the measured read noise, so I think the associated `verify` test is correct as well.

Reply:
This is true if the noise evolution can be approximated by a linear function. But I presume the deviation from the nominal noise starts to evolve in the middle of the flux. 
I'd rather compare the maximum deviation with the n times `mad_std( noiseList )` or compare the number of noises that are deviated from `mad_std` and an expected number from the Gaussian distribution. 
This might not work for some amplifiers that have multimodal behavior in noises though... But we could relax the limit if that's the case.

Reply:
see the comment below.